<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vuyo-HP" ignored="1" name="Default test" tests="1" failures="1" timestamp="27 Jun 2019 09:51:03 GMT" time="0.033" errors="0">
  <testcase name="testLoginAdmin" time="0.0" classname="pages.LoginAdmin.TestLoginAdmin">
    <failure type="org.testng.TestNGException" message="
Method public void pages.LoginAdmin.TestLoginAdmin.testLoginAdmin(java.lang.String,java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : login_data">
      <![CDATA[org.testng.TestNGException: 
Method public void pages.LoginAdmin.TestLoginAdmin.testLoginAdmin(java.lang.String,java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : login_data
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:336)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:498)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1020)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1110)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testLoginAdmin -->
  <testcase name="testLoginAdmin2" time="0.0" classname="pages.LoginAdmin.TestLoginAdmin">
    <ignored/>
  </testcase> <!-- testLoginAdmin2 -->
</testsuite> <!-- Default test -->
